### PR TITLE
Update combobox list-option directive to use text

### PR DESCRIPTION
**Pull request by atmgrifter00**
# Pull Request

## 🤨 Rationale

- #1136

## 👩‍💻 Implementation

Literally just changing `textContent` to `text`.

## 🧪 Testing

Updated a test to show that we can support setting up the content of a `list-option` in more flexible ways. Before the fix, if you were to format your markup in the way that is now in the test, you would likely get unexpected behavior (see comments in linked issue), and the existing tests would fail.

## ✅ Checklist

- [ ] I have updated the project documentation to reflect my changes or determined no changes are needed.

### CODE REVIEW

File: angular-workspace/projects/ni/nimble-angular/src/directives/combobox/tests/nimble-combobox-control-value-accessor.directive.spec.ts [hunk 1]
```diff
@@ -29,8 +29,12 @@ describe('Nimble combobox control value accessor', () => {
         @Component({
             template: `
                 <nimble-combobox #combobox [(ngModel)]="selectedOption" (ngModelChange)="onModelValueChange($event)" [compareWith]="compareWith" [disabled]="selectDisabled" autocomplete="none">
-                    <nimble-list-option *ngFor="let option of selectOptions" [ngValue]="option">{{ option?.name ?? nullValueString }}</nimble-list-option>
-                    <nimble-list-option [ngValue]="dynamicOption">{{ dynamicOption?.name }}</nimble-list-option>
+                    <nimble-list-option *ngFor="let option of selectOptions" [ngValue]="option">
+                        {{ option?.name ?? nullValueString }}
```
Comment (rajsite) [inline angular-workspace/projects/ni/nimble-angular/src/directives/combobox/tests/nimble-combobox-control-value-accessor.directive.spec.ts:33]: Can you add a comment saying that the test intentionally is leaving whitespace around the value. Or maybe a specific separate test where it more explicitly initializes the content of an attribute with extra white space and specifically tests that in the test name.

Main concern is I can easily see a refactoring changing the layout of this fixture and losing coverage of this test case.

**Review by rajsite — APPROVED** [review]

**Review by rajsite — APPROVED** [review]

File: angular-workspace/projects/ni/nimble-angular/src/directives/combobox/tests/nimble-combobox-control-value-accessor.directive.spec.ts (hunk 1 above)
Comment (atmgrifter00) [reply angular-workspace/projects/ni/nimble-angular/src/directives/combobox/tests/nimble-combobox-control-value-accessor.directive.spec.ts]: I've reverted this change, and now made one of the options have whitespace around the main text which both failed some previous tests as well as a new explicit test I've added.
